slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -96,6 +96,16 @@ ::grpc::Status SLAMServiceImpl::GetPositionNew(
 ::grpc::Status SLAMServiceImpl::GetPointCloudMap(
     ServerContext *context, const GetPointCloudMapRequest *request,
     GetPointCloudMapResponse *response) {
+    // if we have a map for localization, use it
```

Comment:
[nit] for consistency with rest of comments

```suggestion
    // If we have a map for localization, use it
```